pkg/controller/manager/types_old.go

```diff
@@ -186,8 +175,12 @@ func (b *BackupTaskSpec) ToManager() (*managerclient.Task, error) {
 	return t, nil
 }
 
-func (b *BackupTaskSpec) ToStatus() *BackupTaskStatus {
-	bts := &BackupTaskStatus{
+func (b *BackupTaskSpec) GetObjectHash() (string, error) {
+	return hashutil.HashObjects(b)
```

Comment:
where do you clear the hash label to have consistent hashing? like in 
https://github.com/scylladb/scylla-operator/blob/b8146506c363b6c1b96cfca55a86f4a426c78090/pkg/resourceapply/helpers.go#L64-L70

Reply:
The hash is calculated from the spec in our API (a wrapper around it, but the point still stands), so setting it in the manager state doesn't affect it. Effectively that also means that an update will only be triggered if the user either a) changes the task spec in our API, or b) modifies the managed hash label value through some other means (e.g. with sctool).

Reply:
I see, this is different because you don't store the hash on the object you compute it from.